projects/RabbitMQ.Client/client/impl/ContentHeaderPropertyWriter.cs

```diff
@@ -45,24 +45,31 @@
 
 namespace RabbitMQ.Client.Impl
 {
-    struct ContentHeaderPropertyWriter
+    internal ref struct ContentHeaderPropertyWriter
```

Comment:
It seems like

- `WriteBit`
- `WriteLong`
- `WriteLonglong`
- `WriteLongstr`
- `WriteShort`

are no longer used. Should we ditch them? Or is it worth keeping them around just in case?

Reply:
I'm fine with ditching them, I left them as they were there before.